src/memories.js

```
@@ -352,6 +352,16 @@ async function generateSceneSummary(mes_id) {
 		debug(settings.hide_scene, memory_history);
 		if (settings.hide_scene) {
 			for (const mes of memory_history) {
+				// Check if this is a comment message (a previous summary)
```

@InspectorCaracal I guess actually looking at this code, the default behavior would always just be to hide everything. This would just not do anything when it encounters an existing note/comment if this is unchecked. So if notes used for other purposes were hidden or visible, they would just stay in their existing state.